apps/meteor/ee/client/voip/modals/DeviceSettingsModal.tsx

```diff
@@ -60,6 +67,11 @@ const DeviceSettingsModal = (): ReactElement => {
 						</Box>
 					</Box>
 				)}
+				{!useIsDeviceManagementEnabled && (
```

Comment:
Do you really want to verify is the hook exists?

Reply:
Nope, fixed!